src/main/scala/dotvisualizer/FirrtlDiagrammer.scala

```diff
@@ -255,3 +255,18 @@ case class Config(
   }
 }
 
+object Config {
+  private val MacPattern = """.*mac.*""".r
+  private val LinuxPattern = """.*n[iu]x.*""".r
+  private val WindowsPattern = """.*win.*""".r
+
+  def getOpenForOs: String = {
+    System.getProperty("os.name").toLowerCase match {
+      case MacPattern()     => "open"
+      case LinuxPattern()   => "xdg-open"
+      case WindowsPattern() => ""          // no clear agreement here.
```

Comment:
Should we use `???` instead of "" here?

Reply:
I don't think so. I want it to run successfully here by default.  I should probably document more how to view if there is no open program